device/iot-device-samples/multiplexing-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ClientManagerBase.java

```diff
@@ -0,0 +1,161 @@
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.device.DeviceTwin.Pair;
```

Comment:
Java has a native Pair object that would be fine here. This Pair class in our library was a bad case of reinventing the wheel, so we shouldn't encourage its use